lib/kennel/models/dashboard.rb

```diff
@@ -103,10 +103,10 @@ def as_json
           title: "#{title}#{LOCK}",
           description: description,
           template_variables: render_template_variables,
-          template_variable_presets: template_variable_presets,
           widgets: all_widgets
         }
 
+        @json[:template_variable_presets] = template_variable_presets if template_variable_presets
```

Comment:
we need to set the nil because otherwise if you remove presets then it would not deleted